### PR TITLE
Logging channel

### LINKED ISSUE #38: add full logging channel support
currently logging works on any channel named "wasteland", i want to make it togglable per server and to be able to choose a specific channel to post logs on

## Commands:
- `!logging` Shows if logging is enabled, and channel name if enabled
- `!logging disable` Disables logging
- `!logging channel <channelName>` Sets logging channel

### PR DESCRIPTION
**Pull request by Hiviexd**
implemented full functionality of logging channel

Closes #38

### CODE REVIEW

File: commands/management/logging.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import MissingPermissions from "./../../data/embeds/MissingPermissions";
+import { ownerId } from "../../config.json";
+import * as database from "./../../database";
+
+export default {
+    name: "logging",
+    description: "Configure the logging system",
+    syntax: "!logging `<action>` `<value>`",
+    example: "!logging `channel` `wasteland`\n!logging `disable`",
+    subcommands: [],
```
Comment (Sebola3461) [inline commands/management/logging.ts:11]: Remove this

File: commands/management/logging.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                return message.channel.send(
+                    "❗ Invalid action. Provide a valid action."
+                );
+
+            if(params.action == "disable") {
+                guild.logging.enabled = false;
+
+                await database.guilds.updateOne(
+                    { _id: message.guildId },
+                    { $set: { logging: guild.logging } }
```
Comment (Sebola3461) [inline commands/management/logging.ts:84]: you can change this line to `{ logging: guild.logging }`

**Review by Sebola3461 — CHANGES_REQUESTED** [review]

File: commands/management/logging.ts (hunk 1 above)
Comment (Hiviexd) [reply commands/management/logging.ts]: oops

File: commands/management/logging.ts (hunk 2 above)
Comment (Hiviexd) [reply commands/management/logging.ts:83]: ok

**Review by Sebola3461 — APPROVED** [review]

**Pull request merged**